vmngclient/api/administration.py

```diff
@@ -133,3 +143,59 @@ def enable_cloud_on_ramp_for_saas_mode(self, disable=False):
         data = {"mode": "on"} if not disable else {"mode": "off"}
         response = self.session.put(url_path, data)
         return True if response.status_code == 200 else False
+
+    def get_organization_name(self) -> List[str]:
+        endpoint = "/dataservice/settings/configuration/organization"
+
+        return [organizations_info.get("org") for organizations_info in self.session.get_data(endpoint)]
```

Comment:
Some of these getters can be refactored . We can use enums to pick up the scope and use the same query mechanism

Reply:
Implemented update, in separate PR will add getter.